auxiliary/x_aux_lib.c

```diff
@@ -380,7 +380,7 @@ void GEAD_LIBSTR(int m, int n, REAL alpha, struct MAT *sA, int ai, int aj, struc
 
 
 // set all elements of a strmat to a value
-void GESE(int m, int n, REAL alpha, struct MAT *sA, int ai, int aj)
+void GESE_LIBSTR(int m, int n, REAL alpha, struct MAT *sA, int ai, int aj)
```

Comment:
why?

Reply:
To be consistent with the naming. But actually the naming convention for macro seems still in the process to be updated.
To recap:
- `GESE` style is used for functions with blas interface. I.e:
`#define COPY dcopy_`  in blasfeo_api/d_blas2.c
`void GEMM(`  in blas_api/xgemm_ref.c

- `GESE_LIBSTR` style for  blasfeo interface. I.e:
`#define TRMV_LNN_LIBSTR blasfeo_dtrmv_lnn` in blasfeo_api/d_blas2_lib.c 
But is not used in blasfeo_api/x_blas3_lib.c where `*_LIBSTR`, `STRMAT` are dropped in favor of `GEMM_NT` (routine name plus blas hard-coded flags), `XMAT`.

The plan is to make the blas3 current style the default naming convention for macro? 
In this case I propose to use `BLASFEO_GEMM_NT` for function with "blasfeo" interface, or in general having some way to distinguish a BLASFEO macro even with there are no hard-coded flags i.e. `GESE_`

Reply:
Yes the plan was to adopt the blas3 naming convention as the default one. None of the macro name is exposed, so it is not such a high priority. Is it even necessary to distinguish between BLAS and BLASFEO API in the macro names?